package.json

```diff
@@ -0,0 +1,71 @@
+{
+  "name": "coffechat",
+  "version": "0.0.0",
+  "description": "Connect Mentors and Mentees by making the process of connecting and then scheduling appointments as easy as possible.",
+  "main": "index.js",
+  "scripts": {
+    "test": "jest",
+    "dev": "ts-node-dev --no-deps --respawn  --no-notify --ignore-watch '.next' --project tsconfig.server.json src/server/index.ts",
+    "eslint": "eslint .",
+    "eslint:fix": "eslint . --quiet --fix",
+    "stylelint": "stylelint '**/*.tsx'",
+    "stylelint:fix": "stylelint '**/*.tsx' --fix"
```

Comment:
I'd imagine we probably need `prettier` and `prettier:fix` commands here as well, no?

Reply:
So, the way this is set up, `esllint` will run prettier as part `eslint` and `eslint:fix`. So we're good on the javascript files.

For the markdown, yaml, etc. I was not planning/requiring on running prettier, but that's useful.  do you think I should do this?

Reply:
Gotcha. I personally like using Prettier on markdown in particular